components/Navbar.js

```diff
@@ -3,49 +3,68 @@ import React from 'react';
 
 import { BsSearch } from 'react-icons/bs';
 import { FaRegHeart } from 'react-icons/fa';
+import { GiHamburgerMenu } from 'react-icons/gi';
+import { ImCross } from 'react-icons/im';
+
+import SideNavbar from './SideNavbar';
 
 export default function Navbar() {
+  const [showSideNav, setShowSideNav] = React.useState(false);
   return (
-    <div className="navbar mb-2 shadow-lg bg-neutral text-neutral-content justify-center sticky top-0 z-50">
-      <div className="flex-none px-2 mx-2">
-        <span className="text-lg font-bold">
-          {'Margarita Humanitarian Foundation'}
-        </span>
-      </div>
-      <div className="flex-1 px-2 mx-2 hidden md:flex">
-        <div className="items-stretch hidden lg:flex">
-          <Link href="/">
-            <a className="btn btn-ghost btn-sm rounded-btn">{'Home'}</a>
-          </Link>
-          <Link href="/in-kind">
-            <a className="btn btn-ghost btn-sm rounded-btn">{'In-Kind'}</a>
-          </Link>
-          <Link href="/fund">
-            <a className="btn btn-ghost btn-sm rounded-btn">{'Fund'}</a>
-          </Link>
-          <Link href="/give-your-time">
-            <a className="btn btn-ghost btn-sm rounded-btn">
-              {'Give Your Time'}
-            </a>
-          </Link>
-          <Link href="/partner-with-us">
-            <a className="btn btn-ghost btn-sm rounded-btn">
-              {'Partner With Us'}
-            </a>
-          </Link>
-        </div>
-      </div>
-      <div className="flex-none hidden sm:flex">
-        {'Help Families in Need'}
-        <button className="btn btn-square btn-ghost" type="button">
-          <FaRegHeart className="inline-block w-5 h-5 stroke-current" />
-        </button>
-      </div>
-      <div className="flex-none hidden sm:flex">
-        <button className="btn btn-square btn-ghost" type="button">
-          <BsSearch className="inline-block w-5 h-5 stroke-current" />
+    <>
+      <SideNavbar setShow={setShowSideNav} show={showSideNav} />
+      <div className="navbar mb-2 shadow-lg bg-neutral text-neutral-content justify-center sticky top-0 z-50">
+        <button
+          className=" btn-ghost flex  lg:hidden"
+          onClick={() => setShowSideNav(!showSideNav)}
+          type="button"
+        >
+          {showSideNav ? (
+            <ImCross className="inline-block w-5 h-5 stroke-current " />
+          ) : (
+            <GiHamburgerMenu className="inline-block w-5 h-5 stroke-current " />
+          )}
         </button>
+        <div className="flex-none px-2 mx-2">
+          <span className="text-lg font-bold">
+            {'Margarita Humanitarian Foundation'}
```

Comment:
Again, things are looking great, @akshitarora921. Here I noticed that certain sizes display the hamburger menu incorrectly due to the text pushing it out of view. Maybe at smaller resolutions the text on line 31 can be smaller to fix this. 
<img width="315" alt="Screen Shot 2021-10-04 at 4 32 28 PM" src="https://user-images.githubusercontent.com/68655112/135939063-c49e7c28-d8c3-4dd9-8e2b-886671618e56.png">
<img width="338" alt="Screen Shot 2021-10-04 at 4 33 17 PM" src="https://user-images.githubusercontent.com/68655112/135939088-52585f5b-b35e-46d1-bbc3-483ca252c329.png">